<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-MIL7REQD" ignored="0" name="Test" tests="4" failures="1" timestamp="2020-05-26T01:59:34 EDT" time="0.065" errors="0">
  <testcase name="test2" time="0.008" classname="tests.TestNGTest"/>
  <testcase name="test1" time="0.001" classname="tests.TestNGTest"/>
  <testcase name="test1" time="0.002" classname="tests.TestNGTest"/>
  <testcase name="test3" time="0.003" classname="tests.TestNGTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at tests.TestNGTest.test3(TestNGTest.java:37)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- Test -->
